safe/gis/expressions.py

```diff
@@ -39,7 +39,7 @@ def inasafe_impact_analysis_layer(field, feature, parent):
 
     index = layer.fieldNameIndex(field)
     if index < 0:
-        return None
+        return index
```

Comment:
The code won't works if we give Null or None as a parameter and will always returns None or Null. Seems like they don't recognize Null. Could you give it a try?